exchanges/bitfinex/bitfinex_test.go

```diff
@@ -1085,7 +1085,7 @@ func TestWsAuth(t *testing.T) {
 	runAuth(t)
 }
 
-func runAuth(t *testing.T) {
+func runAuth(t *testing.T) { //nolint
```

Comment:
Please be more specific with your nolint usage. Here is an example from our codebase:
`//nolint:forcetypeassert // Not necessary from a pool`
we specify what we are nolinting and we provide a comment on why we are nolinting